Dockerfile

```diff
@@ -1,17 +1,20 @@
-ARG CP_KAFKA_CONNECT_BASE_VERSION=6.0.1
+ARG CP_KAFKA_CONNECT_BASE_VERSION=7.2.2
 
 FROM confluentinc/cp-kafka-connect-base:$CP_KAFKA_CONNECT_BASE_VERSION
 
 ENV COMPONENT=kafka-connect
+ENV CONNECT_PLUGIN_PATH=/usr/share/java,/usr/share/confluent-hub-components,/usr/share/filestream-connectors
 
-ARG CONFLUENTINC_JDBC_CONNECTOR_VERSION=10.0.1
-ARG CONFLUENTINC_HDFS_CONNECTOR_VERSION=10.0.0
-ARG CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION=11.0.0
-ARG CONFLUENTINC_S3_CONNECTOR_VERSION=5.5.3
-ARG CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION=5.5.3
+ARG CONFLUENTINC_JDBC_CONNECTOR_VERSION=10.5.4
+ARG CONFLUENTINC_HDFS_CONNECTOR_VERSION=10.1.13
+ARG CONFLUENTINC_ELASTICSEARCH_CONNECTOR_VERSION=14.0.2
+ARG CONFLUENTINC_S3_CONNECTOR_VERSION=10.2.2
+ARG CONFLUENTINC_AVRO_CONVERTER_CONNECTOR_VERSION=7.2.2
```

Comment:
Do we know if anything changed with these since the previous versions? Do we have any kind of example anywhere for these? Because if not then I guess my question hardly matters, but if we have examples we should update those as well if needed.

Reply:
There is an example for the S3 sink connector in the documentation. 
This PR is only update the connectors there is another jira ticket which cover the testing